site/docs/numbers/porting/port-ins/bulkPortIns.mdx

```diff
@@ -0,0 +1,309 @@
+			<Description>TN list contains at least one toll free number that cannot be ported due to spare status.</Description>
+			<TnList>
+				<Tn>8005587721</Tn>
+			</TnList>
+		</Error>
+		<Error>
+			<Code>7643</Code>
+			<Description>TN list contains at least one toll free number that cannot be ported due to unavailable status.</Description>
+			<TnList>
+				<Tn>8335098835</Tn>
+			</TnList>
+		</Error>
+	</ErrorList>
+</TnListResponse>
+```
+
+In the above scenario, all telephone numbers will be accounted for in the PortableTnList or an Error element. For details on toll free number validation, please see [Toll Free Porting Validations](/docs/numbers/porting/port-ins/stand-alone-portins/tollFreePortingValidations).
+
+Sample GET /tnList response when validation cannot be completed.
+
+```xml
+<TnListResponse>
+    <ErrorList>
+        <Error>
+            <Code>7626</Code>
+            <Description>No results from toll free porting vendor after trying for 300 seconds.</Description>
+            <TelephoneNumbers>
+                <Tn>8336001666</Tn>
+            </TelephoneNumbers>
+        </Error>
+    </ErrorList>
+    <OrderId>46602764-7dc0-4958-bc09-2c9e20f59bbc</OrderId>
+    <PortableTnList/>
+    <ProcessingStatus>INVALID_DRAFT_TNS</ProcessingStatus>
+</TnListResponse>
+```
+
+In the above scenario, the system has retried for 5 minutes before giving up. You can kick off another 5 minutes of retry attempts (and hopefully succeed!) by issuing a PATCH operation to [/accounts/{accountId}/bulkPortins/{orderId}](/apis/numbers/#operation/PatchBulkOrder) with a payload that includes the RetryValidation element with a value of true. If this condition persists, please contact Bandwidth for assistance. We may have information about toll free porting vendor scheduled maintenance, or estimated time to repair for unplanned outages.
+
+Sample GET /tnList response with portable numbers.
+
+```xml
+<TnListResponse>
+    <ChildPortinOrderList>
+        <ChildPortinOrder>
+            <OrderId>8762dd42-1f75-4774-93ca-70b3d148ce09</OrderId>
+            <TnList>
+                <TN>8336532076</TN>
+            </TnList>
+        </ChildPortinOrder>
+        <ChildPortinOrder>
+            <OrderId>6691fabd-4c0e-48f5-849d-5a11e2659e21</OrderId>
+            <TnList>
+                <TN>8336531993</TN>
+            </TnList>
+        </ChildPortinOrder>
+    </ChildPortinOrderList>
+    <OrderId>b3d89f9e-a46e-4d56-aaad-9c9d8ac98bb9</OrderId>
+    <PortableTnList>
+        <TN>8336531993</TN>
+        <TN>8336532076</TN>
+    </PortableTnList>
+    <ProcessingStatus>VALID_DRAFT_TNS</ProcessingStatus>
+</TnListResponse>
+```
+
+In the above scenario, both of the toll free numbers were portable. The two toll free numbers are added to separate child port-in orders because they belong to different controlling RespOrgs and therefore cannot be ported together.
+
+# Updating the Bulk Port-in (Optional)
+
+## Updating Bulk Port-in Data Elements
```

Comment:
Similar to the formatting issue in the overview page. This looks weird since the two headings are getting formatted to the same font size. It makes it look like it's an empty section followed by a different section rather than this being a subheading.
![image](https://user-images.githubusercontent.com/76067287/222838370-bf86597c-2f9a-4e19-a075-96f3be4feff8.png)